features/support/object_model_pages/ivl/iap/ivl_iap_other_income_page.rb

```diff
@@ -4,7 +4,7 @@
 class IvlIapOtherIncomePage
 
   def self.has_unemployment_income_yes_radiobtn
-    '#has_unemployment_income_true'
+    'has_unemployment_income_true'
```

Comment:
missing the #

Reply:
it's not required when using Choose method from capybara when working with radio buttons